helper/impls/helper-impl-110/src/main/java/com/linkedin/avroutil1/compatibility/avro110/FieldBuilder110.java

```diff
@@ -98,7 +98,7 @@ public FieldBuilder copyFromField() {
     } catch (Exception e) {
       throw new IllegalArgumentException("unable to convert default value " + _defaultVal + " into something avro can handle", e);
     }
-    Schema.Field result = new Schema.Field(_name, _schema, _doc, avroFriendlyDefault, _order);
+    Schema.Field result = new Schema.Field(_name, _schema, _doc, avroFriendlyDefault, _order == null ? Order.ASCENDING : _order);
```

Comment:
nit - if we initialize the _order field to ASCENDING in the parent class it would save us having to do this in all the subclasses ?

Reply:
(we would need to add an assert that _order != null at this point though)

Reply:
...which maybe we can also do in the parent class by introducing some validate() method called 1st thing on build()

Reply:
I wanted to do it this way so that even if somebody explicitly calls `setOrder(null)`, we'll correctly transform it into ASCENDING.

Reply:
Also, there's no "parent class". Each of the FieldBuilderXX implements an interface, and don't extend some parent class.

Reply:
Changed the code. PTAL.